website/docs/introduction/push-notifications.md

```diff
@@ -0,0 +1,47 @@
+---
+id: push-notifications
+title: Push Notifications
+---
+
+A push notification _is a message that is "pushed" from backend server or application to user interface, e.g. 
+(but not limited to) mobile applications and desktop applications_(see [wikipedia](https://en.wikipedia.org/wiki/Push_technology#Push_notification)).
+
+Push notification main advantage is to keep user interest high and avoid your application getting forgotten and uninstalled 
+by allowing  you to send unsolicited messages (notifications) to the users that have your app installed.
+
+:::warning
+Push notifications are great marketing tool, but users could mute you if you abuse them.
+:::
+
+## _Push notification_ vs _text messages_
+
+One question that could arise could be: _"Ok, if push notifications are a way to send unsolicited messages to the user, 
+what is the difference with text messages?"_
+
+Although they are similar in many ways, they have some important difference:
+* you don't need to know the phone number of the user to send push notifications
+* unsolicited SMS messages are illegal in some [state](https://www.consumer.ftc.gov/articles/how-recognize-and-report-spam-text-messages)
+* _Push Notifications_ are strictly related to your application, while text messages aren't. When the user taps on a push 
+  notification, your app will open. When the user taps on a text message, the messaging app will open 
+* the user can choose to enable or disable _push notification_ from a particular app. That's not true for text messages 
+  (user have to **block** you to stop receiving your messages)
```

Comment:
Put in a not here that push notification can trigger your application to do work and text messages can't